### PR TITLE
Finished seed for demo

### PR DESCRIPTION
**Pull request by DenzelBraithwaite**
Creates very specific seed that will be used for the demo on Thursday. I've tested it several times but please feel free to fetch the branch and test it locally.

### CODE REVIEW

File: app/assets/cities.json [hunk 1]
```diff
@@ -1,21 +1,30 @@
 [
   "Montreal, Côte-des-Neiges–Notre-Dame-de-Grâce",
+  "Montreal, Côte Saint-Luc",
```
Comment (DenzelBraithwaite) [inline app/assets/cities.json:3]: Adds some extra locations for more pokeball markers on the map

File: app/assets/species.json [hunk 2]
```diff
@@ -81,6 +81,7 @@
   "Metapod",
   "Mew",
   "Mewtwo",
+  "Minun",
```
Comment (DenzelBraithwaite) [inline app/assets/species.json:84]: Adds a few species for demo

File: db/seeds.rb [hunk 3]
```diff
@@ -34,44 +79,38 @@
 #---------------------------------OWNER-------------------------------------#
 #############################################################################
 
+# Prevents Gary from having other plusle and minun pokemons for demo
```
Comment (DenzelBraithwaite) [inline db/seeds.rb:82]: This is for the demo presentation, I don't want duplicate 'plusle' and 'minun' pokemons.

File: db/seeds.rb [hunk 4]
```diff
@@ -233,14 +262,79 @@
...
-  Booking.create!(
+  booking = Booking.create!(
     pokemon_id: pokemon.id,
     user_id: renter.id,
     start_date: Faker::Date.between(from: 15.days.ago, to: 10.days.ago),
     end_date: Faker::Date.between(from: 10.days.ago, to: 2.days.ago)
   )
 end
 
+# Creates specific pokemon to make a past booking with during demo, owner is Gary (pokemon_id: to_confirm?)
```
Comment (DenzelBraithwaite) [inline db/seeds.rb:273]: This basically leaves 3 reviews by fellow classmates ;) for the demo

**Comment by stshayna** [comment]
No conflicts with master - will test it out and push to Heroku! Thanks.

**Pull request merged**